app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/acquisitiondetails/summary.scala.html

```diff
@@ -70,23 +74,32 @@
       )
     } else {
       @cyaRow(
-        messages("acquisitionDetails.cyaLabel.rebased"),
-        Html(messages("generic.yes")),
-        Some(cyaChange(messages("acquisitionDetails.cyaChange.rebasedAcquisitionPrice"), routes.AcquisitionDetailsController.rebasedAcquisitionPrice().url)),
-        "rebasedAcquisitionPrice"
-      )
-      @cyaRow(
-        messages("acquisitionDetails.cyaLabel.rebasedValue"),
+        messages("acquisitionDetails.cyaLabel.rebasedValue", LocalDateUtils.govShortDisplayFormat(cutoffDate)),
         Html(MoneyUtils.formatAmountOfMoneyWithPoundSign(rebasedValue.inPounds())),
         Some(cyaChange(messages("acquisitionDetails.cyaChange.rebasedAcquisitionPrice"), routes.AcquisitionDetailsController.rebasedAcquisitionPrice().url)),
         "rebasedAcquisitionPrice-value"
       )
     }
   }
-  @if(answers.improvementCosts.value === 0L){
+  @if(answers.shouldUseRebase){
     @cyaRow(
-      messages("improvementCosts.title"),
+      messages("shouldUseRebase.title", LocalDateUtils.govShortDisplayFormat(cutoffDate)),
       Html(messages("generic.no")),
+      Some(cyaChange(messages("acquisitionDetails.cyaChange.improvementCosts"), routes.AcquisitionDetailsController.shouldUseRebase().url)),
```

Comment:
`acquisitionDetails.cyaChange.improvementCosts` is not right here - we should create a new one with a dummy value for now. It's what gets read out by a screen reader. It will say "Change x" where x is what we put here. You can say something like `whether you are using the rebased value in your calculation`